extension-manifest-v3/tests/utils/trackerdb.test.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import '../setup.js';
+import { describe, it, beforeEach } from 'node:test';
+import assert from 'node:assert';
+import { isTrusted } from '../../src/utils/trackerdb.js';
+
+describe('/utils/trackerdb.js', () => {
+  describe('#isTrusted', () => {
+    let exception;
+
+    beforeEach(() => {
+      exception = {
+        blocked: false,
+        blockedDomains: ['foo.org'],
```

Comment:
This is a bit of a weird setup, having a domain trusted and blocked at the same time. Is such a case possible to reach in production?